### PR TITLE
alphaerrors - doubt - shape

### PR DESCRIPTION
**Pull request by mardeke**

### CODE REVIEW

**Review by theroggy — COMMENTED** [review]
I made some minor changes, I think it is good to go now. Can you have a look if you agree?

**Comment by theroggy** [comment]
@JoeriBroeckx do you think the output is OK for parcelinspector?

File: cropclassification/postprocess/classification_postprocess.py [hunk 1]
```diff
@@ -94,7 +99,12 @@ def calc_top3_and_consolidation(input_parcel_filepath: Path,
     pred_df[conf.columns['prediction_cons_status']].fillna('NOK', inplace=True)    
 
     logger.info("Write full prediction data to file")
-    pdh.to_file(pred_df, output_predictions_filepath)
+    pdh.to_file(pred_df, output_predictions_filepath)  
+
+    # Output to geo file
+    input_parcel_gdf = gfo.read_file(input_parcel_geofilepath)
```
Comment (JoeriBroeckx) [inline cropclassification/postprocess/classification_postprocess.py:105]: Something that we should look out for.. I think we read the parcel input file 3-4 times during the prediction process. This causes a delay.

**Comment by JoeriBroeckx** [comment]
> @JoeriBroeckx do you think the output is OK for parcelinspector?

Sure, it just requires a geometry file to work. Further filtering can happen in the script itself.

File: cropclassification/helpers/geofile.py [hunk 2]
```diff
@@ -10,6 +10,8 @@
 import fiona
 import geopandas as gpd
 
+from cropclassification.helpers import pandas_helper as pdh
```
[the remark this answers is not in the document]
Comment (theroggy) [reply cropclassification/helpers/geofile.py:13]: The ``pandas_helper`` will stay in use, it's the ``geofile`` helper that's deprecated ;-).

File: cropclassification/postprocess/classification_postprocess.py (hunk 1 above)
Comment (theroggy) [reply cropclassification/postprocess/classification_postprocess.py:105]: Yes, it's the way cropclassification works in general. Everything is passed through as files from function to function to be sure the intermediate result are properly written... but this indeed gives delays, especially for geo files. 

Once geofileops uses pyogrio this delay will become a lot smaller, but it's definitely also a valid question whether the files being reread every time is really needed... Anyway not something to be adressed in this PR.

**Pull request merged**